config/sillytips.txt

```diff
@@ -8,6 +8,9 @@ Just like real life the entropy of the game can only increase with time. If thin
 Completing your objectives is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
 The more obscure and underused a game mechanic is, the less likely your victims are to be able to deal with it.
 Space is cold and it will quickly freeze you to death if you don't protect yourself. This isn't how thermodynamics really works but just go with it.
+Blobs are weak to fire! Use a flame thrower for maximum damage!
```

Comment:
This is also actually true; blobs can only block fire with shield blobs, and those can be weakened to let fire pass.